src/features/common/Layout/CustomModal/index.tsx

```diff
@@ -6,10 +6,11 @@ import styles from './CustomModal.module.scss';
 interface Props {
   isOpen: boolean;
   handleContinue: () => void;
-  handleCancel: () => void;
+  handleCancel?: () => void;
   buttonTitle: string;
   modalTitle: string;
   modalSubtitle: string;
+  isCancel: boolean;
```

Comment:
Is `isCancel` actually needed?

You can check whether `handleCancel` is undefined instead.

If you choose to keep this, rename `isCancel` to a more relevant name e.g. `isCancellable`, or `shouldAllowCancel` ?